thirdparty/djvulibre/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ ExternalProject_Add(
     DOWNLOAD_COMMAND ${CMAKE_COMMAND} -P ${GIT_CLONE_SCRIPT_FILENAME}
     PATCH_COMMAND NOCONFIGURE=1 ./autogen.sh
     BUILD_IN_SOURCE 1
-    CONFIGURE_COMMAND sh -c ${CFG_CMD_STR} COMMAND sed -i "s|-lstdc++||g" libtool
+    CONFIGURE_COMMAND sh -c ${CFG_CMD_STR} COMMAND sed -ie "s|-lstdc++||g" libtool
```

Comment:
Yeah, `sed -ie` should work on both BSD and Linux but on both systems there will be a backup file left. I just cannot find a comman parameter to do in-place replace on both systems. Probably we should have a `SED` command that is `sed -i` on linux and `sed -i ''` on darwin.